index.html

```diff
@@ -0,0 +1,173 @@
+        </div>
+      </nav>
+    </header>
+      
+    <main class="my-main">
+      <!-- headline section -->
+      <section id="my-headline" class="my-headline">
+        <div class="my-headline-container">
+          <p class="my-hello">"Hello Developers!"</p>
+          <div class="my-headline-header-container">
+            <h1 class="my-headline-header">TEHRAN<br>CODERCONF<br>2022</h1>
+          </div>
+          <p class="my-headline-description">Meet developers from all over the
+            world and reach the ultimate level of knowledge and innovation in web development.</p>
+          <h2 class="my-headline-date">2022.05.12(SAT) ~ 15(THU)</h2>
+          <p class="my-headline-place">@ Tehran CoderConf, Tehran</p>
+        </div>
+      </section>
+
+      <!-- program section -->
+      <section id="my-main" class="my-main-section">
+        <h2 class="my-main-header">Main Program</h2>
+        <hr class="small-hr" />
+        
+        <div class="my-main-section-program">
+          <article class="my-main-program-act">
+            <i class="fas fa-chalkboard fa-3x"></i>
+            <h3 class="my-program-act">Lecture</h3>
+            <p class="my-program-act-description">Listen to the speakers from various countries about the messages of sharing and opening.</p>
+          </article>
+          <article class="my-main-program-act">
+            <i class="fas fa-users fa-3x"></i>
+            <h3 class="my-program-act">Exhibition</h3>
+            <p class="my-program-act-description">Appreciate various creations expositors about the latest technology anouncements.</p>
+          </article>
+          <article class="my-main-program-act">
+            <i class="fas fa-bullhorn fa-3x"></i>
+            <h3 class="my-program-act">Forum</h3>
+            <p class="my-program-act-description">Have the time to listen and share your thought and opinions with experts for each topic.</p>
+          </article>          
+          <article class="my-main-program-act">
+            <i class="fa fa-microphone"></i>
+            <h3 class="my-program-act">Workshop</h3>
+            <p class="my-program-act-description">Try creating your own work using open source license rather than just watching at it.</p>
+          </article>
+          <article class="my-main-program-act">
+            <i class="fa fa-grav fa-3x"></i>
+            <h3 class="my-program-act">Ignite</h3>
+            <p class="my-program-act-description">Get opportunities to network with developers around the world, also after the summit.</p>
+          </article>
+        </div>
+        <button type="button" class="my-main-program-btn">Join The Web Week Dev!</button>
+      </section>
+
+      <!-- speakers section -->
+      <section id="speakers" class="my-speakers-section">
+        <h2 class="my-speakers-header">Featured Speakers</h2>
+        <hr class="small-hr" />
+        <div class="my-speakers-group"></div>
+      </section>
+    </main>
```

Comment:
- [ ] This section should be divided into two columns in the desktop view. When I click on the `see more` button, that page does not appear like that on the given design. Kindly take a look at the screenshot below as fix it as suggested by the previous reviewer.
![Screenshot from 2022-02-12 12-43-59](https://user-images.githubusercontent.com/20245164/153707439-707ce385-3b8a-41ac-a769-f8defd2db64d.png)

Reply:
Thanks for your attention, but in my template, It is as I created

Reply:
https://www.behance.net/gallery/29845175/CC-Global-Summit-2015

Reply:
in My webpage is in two columns too

Reply:
I think it's because your browser zooming setting that is not set on 100%

Reply:
Kindly check again, I have tried even with different browser and its the same

Reply:
- [ ] [OPTIONAL] I can see that the section has been divided into two columns, however, the `more` button is not supposed to be seen on the desktop view. Kindly remove the button and display all the speakers.